packages/jest-snapshot/package.json

```diff
@@ -33,11 +33,17 @@
     "jest-haste-map": "^24.9.0",
     "prettier": "^1.13.4"
   },
+  "peerDependencies": {
```

Comment:
add `peerDependenciesMeta` and mark them as optional?

Reply:
@SimenB updated, though I'm not familiar with peerDependenciesMeta - is the below what you meant?

Reply:
yup 👍

Reply:
this seems to have disappeared?

Reply:
re-adding in merge